src/components/agent-inbox/hooks/use-interrupted-actions.tsx

```diff
@@ -369,27 +391,25 @@ export default function useInterruptedActions<
     updateQueryParams(VIEW_STATE_THREAD_QUERY_PARAM);
   };
 
-  const supportsMultipleMethods =
-    humanResponse.filter(
-      (r) => r.type === "edit" || r.type === "accept" || r.type === "response"
-    ).length > 1;
-
   return {
     handleSubmit,
     handleIgnore,
     handleResolve,
-    humanResponse,
     streaming,
     streamFinished,
     currentNode,
     loading,
-    threadId: threadData.thread.thread_id,
-    isIgnoreAllowed: !!humanResponse.find((r) => r.type === "ignore"),
-    supportsMultipleMethods,
+    threadId: threadData?.thread.thread_id || "",
+    isIgnoreAllowed: threadData?.interrupts?.[0]?.config.allow_ignore || false,
+    supportsMultipleMethods: !!(
+      threadData?.interrupts?.[0]?.config.allow_respond &&
+      threadData?.interrupts?.[0]?.config.allow_edit
+    ),
```

Comment:
Can we keep the same logic as before?
```suggestion
    supportsMultipleMethods: humanResponse.filter(
      (r) => r.type === "edit" || r.type === "accept" || r.type === "response"
    ).length > 1;
```